src/layouts/Docs.tsx

```diff
@@ -1,3 +1,288 @@
+  Heading4 as MdHeading4,
+  Paragraph,
+} from "@/components/MdComponents"
+
+// Utils
+import { EDIT_CONTENT_URL } from "@/lib/constants"
+import { DocsFrontmatter, MdPageContent } from "@/lib/interfaces"
+
+const Page = (props: ChildOnlyProp & Pick<FlexProps, "dir">) => (
+  <Flex
+    direction="column"
+    w="full"
+    borderBottom="1px"
+    borderColor="border"
+    {...props}
+  />
+)
+
+const ContentContainer = (props: ChildOnlyProp) => (
+  <Flex
+    justify={"space-between"}
+    w="full"
+    py={0}
+    pl={0}
+    pr={{ base: 0, lg: 8 }}
+    backgroundColor="ednBackground"
+    {...props}
+  />
+)
+
+const baseHeadingStyle: HeadingProps = {
+  fontFamily: "mono",
+  textTransform: "uppercase",
+  fontWeight: "bold",
+}
+
+const H1 = (props: HeadingProps) => (
+  <MdHeading1
+    {...baseHeadingStyle}
+    fontSize={{ base: "2rem", md: "2.5rem" }}
+    mt={{ base: 0, md: 8 }}
+    mb={{ base: 4, md: 8 }}
+    {...props}
+  />
+)
+
+const H2 = (props: HeadingProps) => (
+  <MdHeading2
+    {...baseHeadingStyle}
+    fontSize="2xl"
+    lineHeight={{ base: 1.2, md: 1.4 }}
+    pb={2}
+    borderBottom="1px"
+    borderColor="border"
+    {...props}
+  />
+)
+
+const baseSubHeadingStyles: HeadingProps = {
+  lineHeight: 1.4,
+  fontWeight: "semibold",
+}
+
+const H3 = (props: HeadingProps) => (
+  <MdHeading3
+    {...baseSubHeadingStyles}
+    mt={12}
+    {...props}
+  />
+)
+
+const H4 = (props: HeadingProps) => (
+  <MdHeading4
+    {...baseSubHeadingStyles}
+    {...props}
+  />
+)
+
+const UnorderedList = (props: ListProps) => (
+  <ChakraUnorderedList ms="1.45rem" {...props} />
+)
+const OrderedList = (props: ListProps) => (
+  <ChakraOrderedList ms="1.45rem" {...props} />
+)
+
+const ListItem = (props: ListItemProps) => (
+  <ChakraListItem color="text300" {...props} />
+)
+
+const Divider = () => (
+  <ChakraDivider
+    my={16}
+    w="10%"
+    borderBottomWidth={1}
+    borderColor="homeDivider"
+  />
+)
+
+// Apply styles for classes within markdown here
+const Content = (props: ChildOnlyProp) => {
+  const mdBreakpoint = useToken("breakpoints", "md")
+
+  return (
+    <Box
+      as="article"
+      flex={`1 1 ${mdBreakpoint}`}
+      maxW={{ base: "full", lg: mdBreakpoint }}
+      pt={{ base: 32, md: 12 }}
+      pb={{ base: 8, md: 16 }}
+      px={{ base: 8, md: 16 }}
+      m="0 auto"
+      sx={{
+        ".featured": {
+          paddingLeft: 4,
+          marginLeft: -4,
+          borderLeft: "1px dotted",
+          borderColor: "primary",
+        },
+        ".citation": {
+          p: {
+            color: "text200",
+          },
+        },
+      }}
+      {...props}
+    />
+  )
+}
+
+const BackToTop = (props: ChildOnlyProp) => (
+  <Flex
+    display={{ lg: "none" }}
+    mt={12}
+    pt={8}
+    borderTop="1px"
+    borderColor="border"
+    {...props}
+  >
+    <Link to="#top">
+      <Translation id="back-to-top" /> ↑
+    </Link>
+  </Flex>
+)
+
+export const docsComponents = {
```

Comment:
We could clean this up a bit since we are already declaring some of these components in the `MdComponents` file. Those are the default components that are included in every md generated page. I'll push a commit.

Reply:
Which components here do you see needing to be cleaned up? When migrating it over, there are some styles only in the developer docs when it comes to headings, and paragraph. I imported from this file, and just extended them with the styles. Feel free to push what you want cleaned up though.

Reply:
I should have clarified. I've already pushed [a commit](https://github.com/ethereum/ethereum-org-fork/pull/114/commits/09c929d617c7d43ab993f35faf67ecb49bd777be) for this, you can check it out and see what I was referring to.

In short, I basically removed some components from the exported mapping that you are doing here because it was redundant. Most of the common components are already defined in the MdComponents file.